### PR TITLE
fix: mfa email template

### PR DESCRIPTION
**Pull request by TorstenDittmann**
## What does this PR do?

- fixes email template for otp
- adds rich text vars to email templates

## Test Plan

- manual

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/config/locale/translations/en.json [hunk 1]
```diff
@@ -4,13 +4,13 @@
     "settings.direction": "ltr",
     "emails.sender": "%s Team",
     "emails.verification.subject": "Account Verification",
-    "emails.verification.hello": "Hey {{user}}",
+    "emails.verification.hello": "Hello {{user}}",
```
Comment (Meldiron) [inline app/config/locale/translations/en.json:7]: If there is no user and you set user to `,`, then you still get `Hello ,` with unwanted space

File: app/controllers/api/account.php [hunk 2]
```diff
@@ -3697,9 +3699,85 @@
                     throw new Exception(Exception::USER_EMAIL_NOT_VERIFIED);
                 }
 
+                $subject = $locale->getText("emails.mfaChallenge.subject");
+                $customTemplate = $project->getAttribute('templates', [])['email.mfaChallenge-' . $locale->default] ?? [];
```
[the remark this answers is not in the document]
Comment (TorstenDittmann) [reply app/controllers/api/account.php:3716]: will be done 👍🏻

File: app/config/locale/templates/email-inner-base.tpl [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<p>{{hello}},</p>
```
Comment (stnguyen90) [inline app/config/locale/templates/email-inner-base.tpl:1]: Don't all the templates need to be updated since this comma is removed?

Comment (TorstenDittmann) [reply app/config/locale/templates/email-inner-base.tpl:1]: Good call, I should actually have the comma the other way around. There was some back and forth with design, so. I sohuld've went the other way.

File: app/controllers/api/account.php [hunk 4]
```diff
@@ -1192,15 +1192,14 @@
 
         $emailVariables = [
             'direction' => $locale->getText('settings.direction'),
-            /* {{user}}, {{team}}, {{redirect}} and {{project}} are required in default and custom templates */
-            'user' => '',
-            'team' => '',
```
Comment (stnguyen90) [inline app/controllers/api/account.php:1197]: team is required. See https://github.com/appwrite/appwrite/pull/6077

File: app/controllers/api/account.php [hunk 5]
```diff
@@ -3697,9 +3712,85 @@
...
+                $emailVariables = [
+                    'direction' => $locale->getText('settings.direction'),
+                    /* {{user}}, {{project}} and {{otp}} are required in the templates */
+                    'user' => $user->getAttribute('name'),
+                    'project' => $project->getAttribute('name'),
+                    'otp' => $code,
+                    'agentDevice' => $agentDevice['deviceBrand'] ?? $agentDevice['deviceBrand'] ?? 'UNKNOWN',
+                    'agentClient' => $agentClient['clientName'] ?? 'UNKNOWN',
+                    'agentOs' => $agentOs['osName'] ?? 'UNKNOWN'
+                ];
```
Comment (stnguyen90) [inline app/controllers/api/account.php:3788]: How will these get exposed for use in the Console in custom templates?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**